src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -248,15 +248,15 @@ class Sidebar extends React.Component {
                   onClick={this.hidePanel}
                   icon={<icons.AvailableWork />}
                   label={t('Available Work')}
-                  to={`/${organization.slug}/tasks/`}
+                  to={`/${organization.slug}/available-work/`}
```

Comment:
I prefer work-definitions as the endpoint name, so that one knows the name of a single unit. We also have a convention that the endpoints are named in plural, which available-work is not.

Reply:
The commit name is confusing ("update task to work unit"). As I understand it, task is the equivalent of work definition, and is not comparable to work unit, which in most cases are "substances".